### PR TITLE
#4792 inputs fixes

### PR DESCRIPTION
**Pull request by desfero**
**Testing notes**
Run #p3

### CODE REVIEW

**Review by arjunu — COMMENTED** [review]
the font size in select seems to have decreased, is that intentional?
image

IO:
image

File: packages/design-system/src/components/buttons/Button.tsx [hunk 1]
```diff
@@ -6,7 +6,7 @@ import { InlineIcon } from "../icons/InlineIcon";
 import { LoadingIndicator } from "../loading-indicator";
 import { ButtonBase } from "./ButtonBase";
 
-import * as styles from "./Button.module.scss";
+import styles from "./Button.module.scss";
```
Comment (arjunu) [inline packages/design-system/src/components/buttons/Button.tsx:9]: why the change from * as to this?

**Comment by arjunu** [comment]
There is a blank item in the confirm your recovery phrase dropdowns, intentional?
image

File: packages/design-system/src/components/buttons/Button.tsx (hunk 1 above)
Comment (desfero) [reply packages/design-system/src/components/buttons/Button.tsx:9]: the newest `style-loader` outputs ES module format by default https://github.com/webpack-contrib/style-loader/blob/master/CHANGELOG.md#-breaking-changes

**Review by arjunu — APPROVED** [review]
